app/views/expenses/new.html.erb

```diff
@@ -0,0 +1,21 @@
+<%= form_with model: [@category, @expense] do |f| %>
+  <article class='form--header'>
+    <%= link_to category_expenses_path(@category) do %>
+      <i class="fa-solid fa-arrow-left"></i>
+    <% end %>
+
+    <h2>Create A Transaction</h2>
+    <div class="actions">
+      <%= f.submit "Create Transaction" %>
+    </div>
+  </article>
+  <% @expense.errors.each do |error| %>
+    <p><%= error.full_message %></p>
+  <% end %>
+  <div class="field">
+    <%= f.text_field :name, autofocus: true, autocomplete: "name", placeholder: 'Name' %>
+  </div>
+  <div class="field">
+    <%= f.number_field :amount, autofocus: true, autocomplete: "amount", placeholder: 'Amount' %>
+  </div>
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you update your forms for both creating the `Category` or `Transaction` by moving the **submit** button bellow the input fields as at the the header submit buttons don't appear to be buttons. This will help users to know where to click after filling the forms. For example you can use the same design given in the template or similar to that
![ksnip_20231121-174148](https://github.com/Akorede3133/expense-insight/assets/83042742/c6645915-eb90-40a0-9d19-9dd3fb193c76)